--- quota-4.09/warnquota.conf.orig	2019-09-12 10:19:47.000000000 +0200
+++ quota-4.09/warnquota.conf	2025-04-14 17:55:14.171234373 +0200
@@ -33,7 +33,7 @@ CC_TO 		= "root@localhost"
 # 12 hours, 5 days)
 # CC_BEFORE = 2 days
 SUPPORT 	= "root@localhost"
-PHONE 		= "123 456 789"
+PHONE 		= "No phone numbers have been defined."
 # Text in the beginning of the mail (if not specified, default text is used)
 # This way text can be split to more lines
 # Line breaks are done by '|' character
