diff -ur quota-3.00.orig/edquota.c quota-3.00/edquota.c
--- quota-3.00.orig/edquota.c	Thu Aug 31 20:35:43 2000
+++ quota-3.00/edquota.c	Wed Oct 25 04:11:09 2000
@@ -100,8 +100,8 @@
 	struct dquot dquot;
 
 	read_dquot(id, &dquot);
-	fprintf(tf, "Used %luKB, limits: soft=%lu hard=%lu\n", (ulong)(dquot.dq_dqb.dqb_curspace>>10),
-	  (ulong)dquot.dq_dqb.dqb_bsoftlimit, (ulong)dquot.dq_dqb.dqb_bhardlimit);
+	fprintf(tf, "Used %luKB, limits: soft=%lu hard=%lu\n", (ulong)qb2kb(dquot.dq_dqb.dqb_curspace),
+	  (ulong)qb2kb(dquot.dq_dqb.dqb_bsoftlimit), (ulong)qb2kb(dquot.dq_dqb.dqb_bhardlimit));
 	fprintf(tf, "Used %lu inodes, limits: soft=%lu hard=%lu\n", (ulong)dquot.dq_dqb.dqb_curinodes,
 	  (ulong)dquot.dq_dqb.dqb_isoftlimit, (ulong)dquot.dq_dqb.dqb_ihardlimit);
 }
@@ -170,8 +170,8 @@
 		fprintf(stderr, "Can't parse limits: %s\n", linebuf);
 		return 1;
 	}
-	dquot.dq_dqb.dqb_bsoftlimit = softlim;
-	dquot.dq_dqb.dqb_bhardlimit = hardlim;
+	dquot.dq_dqb.dqb_bsoftlimit = kb2qb(softlim);
+	dquot.dq_dqb.dqb_bhardlimit = kb2qb(hardlim);
 	if (!fgets(linebuf, MAXLINELEN, tf))
 		goto bad_eof;
 	if (sscanf(linebuf, "Used %lu inodes, limits: soft=%lu hard=%lu", &usage, &softlim, &hardlim) != 3)
