--- quota-3.00/quota.c~	Tue Jul 25 23:36:16 2000
+++ quota-3.00/quota.c	Sun Aug  6 16:48:20 2000
@@ -238,15 +238,15 @@
 	/* If there's no usage or no quotas, inform only in verbose mode */
 	if ((!usage_set || !quota_set) && !(flags & FL_VERBOSE))
 		return;
-	if ((blk.dqb_bhardlimit && ROUNDQUOTABLOCK(blk.dqb_curspace) > blk.dqb_bhardlimit) ||
-	    (blk.dqb_bsoftlimit && ROUNDQUOTABLOCK(blk.dqb_curspace) > blk.dqb_bsoftlimit) ||
+	if ((blk.dqb_bhardlimit && toqb(blk.dqb_curspace) > blk.dqb_bhardlimit) ||
+	    (blk.dqb_bsoftlimit && toqb(blk.dqb_curspace) > blk.dqb_bsoftlimit) ||
 	    (blk.dqb_ihardlimit && blk.dqb_curinodes > blk.dqb_ihardlimit) ||
 	    (blk.dqb_isoftlimit && blk.dqb_curinodes > blk.dqb_isoftlimit))
 		limit_exceed = 1;	/* Limit was exceeded */
 	if (flags & FL_QUIET) {
-		if (ROUNDQUOTABLOCK(blk.dqb_curspace) > blk.dqb_bhardlimit && blk.dqb_bhardlimit)
+		if (toqb(blk.dqb_curspace) > blk.dqb_bhardlimit && blk.dqb_bhardlimit)
 			strcpy(msg, "Over block limit");
-		else if (ROUNDQUOTABLOCK(blk.dqb_curspace) > blk.dqb_bsoftlimit && blk.dqb_bsoftlimit)
+		else if (toqb(blk.dqb_curspace) > blk.dqb_bsoftlimit && blk.dqb_bsoftlimit)
 			strcpy(msg, "Over block quota");
 		if (blk.dqb_curinodes > blk.dqb_ihardlimit && blk.dqb_ihardlimit) {
 			if (msg[0])
@@ -275,7 +275,7 @@
 	else {
 		char buf[MAXTIMELEN];
 		difftime2str(blk.dqb_btime, buf);
-		printf("%8u%c%7u%7u%8s", (uint)ROUNDQUOTABLOCK(blk.dqb_curspace), overlim(qb2kb(ROUNDQUOTABLOCK(blk.dqb_curspace)), blk.dqb_bsoftlimit, blk.dqb_bhardlimit), (uint)qb2kb(blk.dqb_bsoftlimit), (uint)qb2kb(blk.dqb_bhardlimit), buf);
+		printf("%8u%c%7u%7u%8s", (uint)toqb(blk.dqb_curspace), overlim(qb2kb(toqb(blk.dqb_curspace)), blk.dqb_bsoftlimit, blk.dqb_bhardlimit), (uint)qb2kb(blk.dqb_bsoftlimit), (uint)qb2kb(blk.dqb_bhardlimit), buf);
 		difftime2str(blk.dqb_itime, buf);
 		printf("%8u%c%7u%7u%8s\n", (uint)blk.dqb_curinodes, overlim(blk.dqb_curinodes, blk.dqb_isoftlimit, blk.dqb_ihardlimit), (uint)blk.dqb_isoftlimit, (uint)blk.dqb_ihardlimit, buf);
 	}
--- quota-3.00/repquota.c~	Mon Jul 24 18:53:28 2000
+++ quota-3.00/repquota.c	Sun Aug  6 16:49:33 2000
@@ -102,8 +102,8 @@
 	else
 		gid2group(id, name);
 	difftime2str(entry->dqb_btime, time);
-	printf("%-10s%c%c%8d%8d%8d%7s", name, overlim(qb2kb(ROUNDQUOTABLOCK(entry->dqb_curspace)), qb2kb(entry->dqb_bsoftlimit), qb2kb(entry->dqb_bhardlimit)),
-	  overlim(entry->dqb_curinodes, entry->dqb_isoftlimit, entry->dqb_ihardlimit), (int)qb2kb(ROUNDQUOTABLOCK(entry->dqb_curspace)), qb2kb(entry->dqb_bsoftlimit), qb2kb(entry->dqb_bhardlimit), time);
+	printf("%-10s%c%c%8d%8d%8d%7s", name, overlim(qb2kb(toqb(entry->dqb_curspace)), qb2kb(entry->dqb_bsoftlimit), qb2kb(entry->dqb_bhardlimit)),
+	  overlim(entry->dqb_curinodes, entry->dqb_isoftlimit, entry->dqb_ihardlimit), (int)qb2kb(toqb(entry->dqb_curspace)), qb2kb(entry->dqb_bsoftlimit), qb2kb(entry->dqb_bhardlimit), time);
 	difftime2str(entry->dqb_itime, time);
 	printf("%8d%6d%6d%7s\n", entry->dqb_curinodes, entry->dqb_isoftlimit, entry->dqb_ihardlimit, time);
 }
