diff -ur quota-tools.org/warnquota.conf quota-tools/warnquota.conf
--- quota-tools.org/warnquota.conf	Fri Nov  9 21:10:34 2001
+++ quota-tools/warnquota.conf	Wed Apr 10 17:47:10 2002
@@ -1,16 +1,18 @@
-# this is an example warnquota.conf
+# This is an example /etc/warnquota.conf
+# Configured for PLD Linux systems
+
 #
 ; ; and # type comments are allowed
 # and even blank lines
 
 # values can be quoted:
-MAIL_CMD 	= "/usr/my/sendmail/instead/sendmail -t"
-FROM 		= "bas@localhost"
+MAIL_CMD 	= "/usr/sbin/sendmail -t"
+FROM 		= "root@localhost"
 # but they don't have to be:
-SUBJECT 	= Hey, user, clean up your account!
-CC_TO 		= "sysadm@localhost"
+SUBJECT 	= "Disk quota usage on system"
+CC_TO 		= "root"
 SUPPORT 	= "support@myhost.com"
-PHONE 		= "(123) 456-1111 or (222) 333-4444"
+PHONE 		= "No phone numbers have been defined."
 # Text in the beginning of the mail (if not specified, default text is used)
 # This way text can be split to more lines
 # Line break are done by '|' character
