--- quota-tools/quotaon.c.orig	Sun Jun 16 21:00:46 2002
+++ quota-tools/quotaon.c	Mon Jan 13 22:09:24 2003
@@ -177,8 +177,8 @@
 	else if (kernel_formats & (1 << QF_VFSOLD))
 		on = kern_quota_on(mnt->mnt_fsname, type, 1 << QF_VFSOLD) != -1;
 
-	printf("%s quota on %s (%s) is %s\n", type2name(type), mnt->mnt_dir, mnt->mnt_fsname,
-	  on ? "on" : "off");
+	printf(_("%s quota on %s (%s) is %s\n"), type2name(type), mnt->mnt_dir, mnt->mnt_fsname,
+	  on ? _("on") : _("off"));
 	
 	return on;
 }
