diff -ur quota-tools.org/quotastats.c quota-tools/quotastats.c
--- quota-tools.org/quotastats.c	Fri Nov  9 09:07:18 2001
+++ quota-tools/quotastats.c	Wed Apr 10 17:49:24 2002
@@ -101,7 +101,7 @@
 	printf(_("Number of allocated dquots: %ld\n"), (long)dqstats->allocated_dquots);
 	printf(_("Number of free dquots: %ld\n"), (long)dqstats->free_dquots);
 	printf(_("Number of in use dquot entries (user/group): %ld\n"),
-		(long)(dqstats->allocated_dquots - dqstats->free_dquots));
+		(long)(dqstats->free_dquots - dqstats->allocated_dquots));
 	return (0);
 }
 
