diff -ruN quota-tools-romke/repquota.c quota-tools/repquota.c
--- quota-tools-romke/repquota.c	2004-12-30 15:08:45.000000000 +0100
+++ quota-tools/repquota.c	2003-02-14 18:26:40.000000000 +0100
@@ -151,7 +151,7 @@
 	space2str(toqb(entry->dqb_curspace), numbuf[0], flags & FL_SHORTNUMS);
 	space2str(entry->dqb_bsoftlimit, numbuf[1], flags & FL_SHORTNUMS);
 	space2str(entry->dqb_bhardlimit, numbuf[2], flags & FL_SHORTNUMS);
-	printf("%-*s %c%c %7s %7s %7s %6s", PRINTNAMELEN, pname,
+	printf("%-*s %c%c %8s %8s %8s %6s", PRINTNAMELEN, pname,
 	       overlim(qb2kb(toqb(entry->dqb_curspace)), qb2kb(entry->dqb_bsoftlimit), qb2kb(entry->dqb_bhardlimit)),
 	       overlim(entry->dqb_curinodes, entry->dqb_isoftlimit, entry->dqb_ihardlimit),
 	       numbuf[0], numbuf[1], numbuf[2], time);
@@ -242,9 +242,9 @@
 	time2str(h->qh_info.dqi_bgrace, bgbuf, TF_ROUND);
 	time2str(h->qh_info.dqi_igrace, igbuf, TF_ROUND);
 	printf(_("Block grace time: %s; Inode grace time: %s\n"), bgbuf, igbuf);
-	printf(_("                        Block limits                File limits\n"));
-	printf(_("%-9s       used    soft    hard  grace    used  soft  hard  grace\n"), (type == USRQUOTA)?_("User"):_("Group"));
-	printf("----------------------------------------------------------------------\n");
+	printf(_("                         Block limits                 File limits\n"));
+	printf(_("%-9s        used     soft     hard  grace    used  soft  hard  grace\n"), (type == USRQUOTA)?_("User"):_("Group"));
+	printf("-------------------------------------------------------------------------\n");
 
 	if (h->qh_ops->scan_dquots(h, output) < 0)
 		return;
--- quota-tools/po/pl.po.orig	2007-08-05 02:54:47.475278000 +0200
+++ quota-tools/po/pl.po	2007-08-05 09:09:14.959627463 +0200
@@ -2065,13 +2065,13 @@
 
 #: repquota.c:292
 #, c-format
-msgid "                        Block limits                File limits\n"
-msgstr "                        Limity bloków               Limity plików\n"
+msgid "                         Block limits                 File limits\n"
+msgstr "                         Limity bloków                Limity plików\n"
 
 #: repquota.c:293
 #, c-format
-msgid "%-9s       used    soft    hard  grace    used  soft  hard  grace\n"
-msgstr "%-9s       u¿yw.   miêk.   twar. pob³.    u¿yw. miêk. twar. pob³.\n"
+msgid "%-9s        used     soft     hard  grace    used  soft  hard  grace\n"
+msgstr "%-9s        u¿yw.    miêk.    twar. pob³.    u¿yw. miêk. twar. pob³.\n"
 
 #: repquota.c:293
 msgid "User"
--- quota-tools/po/fr.po.orig	2004-09-02 21:38:53.000000000 +0200
+++ quota-tools/po/fr.po	2007-08-05 09:11:30.455348924 +0200
@@ -1661,13 +1661,13 @@
 msgstr "Période de sursis bloc : %s ; période de sursis inode : %s\n"
 
 #: repquota.c:241
-msgid "                        Block limits                File limits\n"
-msgstr "                        Limites bloc               Limites fichier\n"
+msgid "                         Block limits                 File limits\n"
+msgstr "                         Limites bloc                Limites fichier\n"
 
 #: repquota.c:242
 #, c-format
-msgid "%-9s       used    soft    hard  grace    used  soft  hard  grace\n"
-msgstr "%-9s     utilisé souple stricte sursis utilisé souple stricte sursis\n"
+msgid "%-9s        used     soft     hard  grace    used  soft  hard  grace\n"
+msgstr "%-9s      utilisé  souple  stricte sursis utilisé souple stricte sursis\n"
 
 #: repquota.c:242
 msgid "User"
