diff -urp quota-tools-3.12/mntopt.h quota-tools/mntopt.h
--- quota-tools-3.12/mntopt.h	2004-07-21 11:30:49.000000000 -0500
+++ quota-tools/mntopt.h	2004-07-21 11:32:34.443995816 -0500
@@ -12,6 +12,7 @@
 #define MNTTYPE_REISER		"reiserfs"	/* Reiser file system */
 #define MNTTYPE_XFS		"xfs"	/* SGI XFS file system */
 #define MNTTYPE_AUTOFS		"autofs"	/* Automount mountpoint */
+#define MNTTYPE_JFS		"jfs"	/* JFS file system */
 
 /* mount options */
 #define MNTOPT_NOQUOTA		"noquota"	/* don't enforce quota */
diff -urp quota-tools-3.12/quotasys.c quota-tools/quotasys.c
--- quota-tools-3.12/quotasys.c	2004-07-21 11:30:49.000000000 -0500
+++ quota-tools/quotasys.c	2004-07-21 11:34:19.866969088 -0500
@@ -53,6 +53,7 @@ static int correct_fstype(char *type)
 			*next = 0;
 		if (!strcmp(type, MNTTYPE_EXT2) ||
 		    !strcmp(type, MNTTYPE_EXT3) ||
+		    !strcmp(type, MNTTYPE_JFS) ||
 		    !strcmp(type, MNTTYPE_MINIX) ||
 		    !strcmp(type, MNTTYPE_UFS) ||
 		    !strcmp(type, MNTTYPE_UDF) ||
