--- quota-tools/quotastats.c~	Wed May  2 22:56:10 2001
+++ quota-tools/quotastats.c	Wed May  2 22:56:32 2001
@@ -43,7 +43,7 @@
 	fprintf(stdout, _("Number of allocated dquots: %ld\n"), (long)dqstats->allocated_dquots);
 	fprintf(stdout, _("Number of free dquots: %ld\n"), (long)dqstats->free_dquots);
 	fprintf(stdout, _("Number of in use dquot entries (user/group): %ld\n"),
-		(long)(dqstats->allocated_dquots - dqstats->free_dquots));
+		(long)(dqstats->free_dquots - dqstats->allocated_dquots));
 	return (0);
 }
 
