--- quota-3.00/setquota.c~	Tue Jul 25 00:45:25 2000
+++ quota-3.00/setquota.c	Sun Aug  6 18:25:00 2000
@@ -92,8 +92,8 @@
 		dquot.dq_dqb.dqb_itime = cvnum(argstr[optind+2], "Illegal inode grace time.\n");
 	}
 	else {
-		dquot.dq_dqb.dqb_bsoftlimit = cvnum(argstr[optind+1], "Illegal block soft limit.\n");
-		dquot.dq_dqb.dqb_bhardlimit = cvnum(argstr[optind+2], "Illegal block hard limit.\n");
+		dquot.dq_dqb.dqb_bsoftlimit = kb2qb(cvnum(argstr[optind+1], "Illegal block soft limit.\n"));
+		dquot.dq_dqb.dqb_bhardlimit = kb2qb(cvnum(argstr[optind+2], "Illegal block hard limit.\n"));
 		dquot.dq_dqb.dqb_isoftlimit = cvnum(argstr[optind+3], "Illegal inode soft limit.\n");
 		dquot.dq_dqb.dqb_ihardlimit = cvnum(argstr[optind+4], "Illegal inode hard limit.\n");
 		names = optind + 5;
